src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -253,9 +253,20 @@ instance : bounded_order (subgraph G) :=
   le_top := λ x, ⟨set.subset_univ _, (λ v w h, x.adj_sub h)⟩,
   bot_le := λ x, ⟨set.empty_subset _, (λ v w h, false.rec _ h)⟩ }
 
+-- TODO simp lemmas for the other lattice operations on subgraphs
+@[simp] lemma top_adj_iff {v w : V} : (⊤ : subgraph G).adj v w ↔ G.adj v w := iff.rfl
+
+@[simp] lemma top_verts : (⊤ : subgraph G).verts = set.univ := rfl
+
+@[simp] lemma bot_adj_iff : (⊥ : subgraph G).verts = ∅ := rfl
```

Comment:
Names are wrong way round here

Reply:
Still wrong even after merge :/

Reply:
Oops, thanks for catching this! Addressed in #10785